lib/swoosh/adapters/amazon_ses.ex

```diff
@@ -104,12 +104,17 @@ defmodule Swoosh.Adapters.AmazonSES do
   end
 
   defp parse_error_response(body) do
-    node = XMLHelper.parse(body)
+    try do
+      node = XMLHelper.parse(body)
 
-    code = XMLHelper.first_text(node, "//Error/Code")
-    message = XMLHelper.first_text(node, "//Message")
+      code = XMLHelper.first_text(node, "//Error/Code")
+      message = XMLHelper.first_text(node, "//Message")
 
-    %{code: code, message: message}
+      %{code: code, message: message}
+    catch
+      # In case of malformed XML erlang is not throwing error, but is exiting instead
+      :exit, _ -> %{code: 400, message: body}
```

Comment:
I think it would be a bit better if we log out the error and the body, but still re-raise the error. So we know what to fix but don't actually suppress the actual error.